script/deploy/facets/utils/DeployScriptBase.sol

```diff
@@ -39,7 +39,7 @@ contract DeployScriptBase is ScriptBase {
         vm.startBroadcast(deployerPrivateKey);
 
         if (isDeployed()) {
-            console.log("Contract is already deployed");
+            emit log("Contract is already deployed");
```

Comment:
Is there any reason why we use emit rather than console.log?

Reply:
The problem is that solhint doesn’t let us allow console.log in tests/scripts and block it in production files...
That’s why I used emit log instead that we can still log values when needed without breaking this rule.

and as we use `-vvvv --json`during our script run the emitted logs will be always in `logs` field in the json result